spec.html

```diff
@@ -36555,63 +36559,65 @@
-            </li>
-            <li>
-              No other declaration of _f_ that is not a `var` declaration occurs within the function code of _g_
-            </li>
-            <li>
-              There may be occurrences of _f_ as an |IdentifierReference| within the |StatementList| of the |Block| containing the declaration of _f_.
-            </li>
-            <li>
-              There is at least one occurrence of _f_ as an |IdentifierReference| within another function _h_ that is nested within _g_ and no other declaration of _f_ shadows the references to _f_ from within _h_.
-            </li>
-            <li>
-              All invocations of _h_ occur after the declaration of _f_ has been evaluated.
-            </li>
-          </ul>
-        </li>
-        <li>
-          <p>A function is declared and possibly used within a single block but also referenced within subsequent blocks.</p>
-          <ul>
-            <li>
-              A |FunctionDeclaration| whose |BindingIdentifier| is the name _f_ occurs exactly once within the function code of an enclosing function _g_ and that declaration is nested within a |Block|.
-            </li>
-            <li>
-              No other declaration of _f_ that is not a `var` declaration occurs within the function code of _g_
-            </li>
-            <li>
-              There may be occurrences of _f_ as an |IdentifierReference| within the |StatementList| of the |Block| containing the declaration of _f_.
-            </li>
-            <li>
-              There is at least one occurrence of _f_ as an |IdentifierReference| within the function code of _g_ that lexically follows the |Block| containing the declaration of _f_.
-            </li>
-          </ul>
-        </li>
-      </ol>
-      <p>The first use case is interoperable with the semantics of |Block| level function declarations provided by ECMAScript 2015. Any pre-existing ECMAScript code that employs that use case will operate using the Block level function declarations semantics defined by clauses 9, 13, and 14 of this specification.</p>
-      <p>ECMAScript 2015 interoperability for the second and third use cases requires the following extensions to the clause <emu-xref href="#sec-ordinary-and-exotic-objects-behaviours"></emu-xref>, clause <emu-xref href="#sec-ecmascript-language-functions-and-classes"></emu-xref>, clause <emu-xref href="#sec-eval-x"></emu-xref> and clause <emu-xref href="#sec-globaldeclarationinstantiation"></emu-xref> semantics.</p>
-      <p>If an ECMAScript implementation has a mechanism for reporting diagnostic warning messages, a warning should be produced when code contains a |FunctionDeclaration| for which these compatibility semantics are applied and introduce observable differences from non-compatibility semantics. For example, if a var binding is not introduced because its introduction would create an early error, a warning message should not be produced.</p>
+      <emu-note>
```

Comment:
Isn't the `NOTE` formatting redundant? Previous discussion on this topic: https://github.com/tc39/ecma262/pull/341#issuecomment-177268957